charts/oam-core-resources/Chart.yaml

```diff
@@ -18,4 +18,4 @@ version: 0.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: v0.1
+appVersion: v0.3
```

Comment:
@negz @hongchaodeng are we going with `v0.1`?

Reply:
Yeah, let's go with v0.1.